kotlin/android/app/src/main/java/dev/firezone/android/tunnel/TunnelService.kt

```diff
@@ -204,6 +209,27 @@ class TunnelService : VpnService() {
         super.onRevoke()
     }
 
+    // UI updates for resources
+    fun resourcesUpdated() {
```

Comment:
So this is equivalent to `refreshList` or whatever, it's a "something chnaged, fix whatever needs to be fixed" function for the tunnel